scripts/install_submodules.sh

```diff
@@ -78,7 +43,9 @@ stage "Clone submodules"
 
 cd "${PLATFORM_PATH}" && git submodule update --init --recursive
 
-clone_project "${SC_MACHINE_REPO}" "${SC_MACHINE_PATH}" "${SC_MACHINE_BRANCH}" "${SC_MACHINE_COMMIT}"
-clone_project "${SC_WEB_REPO}" "${SC_WEB_PATH}" "${SC_WEB_BRANCH}" "${SC_WEB_COMMIT}"
+"${PLATFORM_PATH}/scripts/install_sc_machine_submodule.sh" "$@"
+"${PLATFORM_PATH}/scripts/install_sc_component_manager_submodule.sh" "$@"
+"${PLATFORM_PATH}/scripts/install_sc_web_submodule.sh" "$@"
+"${PLATFORM_PATH}/scripts/install_ims_kb_submodule.sh" "$@"
```

Comment:
Add, please, warning for installation of ims.ostis.kb

Reply:
There is no alternative, that merged now

Reply:
Why? But you add sc-component-manager

Reply:
sc-component-manager is not deliting ims.ostis.kb. You can install it. The warning should be added after merging ostis-standard specification